src/Adapter/GuzzleFacebookApi.php

```diff
@@ -41,6 +43,14 @@ public function __construct(ClientInterface $client, $redirectUri, $appId, $appS
     }
 
     /** {@inheritdoc} */
+    public function dialog()
```

Comment:
the implementation looks like a "old-PHP" way (`header`, `exit`). Try to use OOP and leverage one of `Response` implementation.